base/src/main/java/io/spine/type/KnownTypes.java

```diff
@@ -137,10 +143,23 @@ public ClassName getClassName(TypeUrl typeUrl) throws UnknownTypeException {
     /**
      * Retrieves all Protobuf types known to the application.
      */
-    public TypeSet getAllTypes() {
+    public TypeSet asTypeSet() {
         return typeSet;
     }
 
+    /**
+     * Obtains known enrichment types.
+     */
+    public ImmutableSet<EnrichmentType> enrichments() {
+        ImmutableSet<EnrichmentType> result =
+                typeSet.messageTypes()
+                       .stream()
+                       .filter(t -> EnrichmentType.test(t.descriptor()))
```

Comment:
Maybe it worth adding a shortcut `test` method to `EnrichmentType` that accepts `MessageType` and not a descriptor? We'll have a cleaner code here with `filter(EnrichmentType::test)`.
 Also, it's the only usage of the `test` method.